src/Std/Time/Duration.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.DateTime
+import Std.Time.Zoned
+
+namespace Std
+namespace Time
+open Internal
+
+set_option linter.all true
+
+/--
+Duration is just a period between two timestamps.
+-/
+def Duration := Timestamp
```

Comment:
Defining `Duration` in terms of `Timestamp` seems very dangerous to me. To me it feels like they happen to contain the same data but are very different semantically, so I would prefer if either
*  `Duration` was defined independently, or
* `Timestamp` was defined in terms of `Duration` (since `Timestamp` is the _duration_ since UNIX epoch).

It should not be possible to take the `neg` of a timestamp and it should not be possible to `add` two timestamps. The functions that do `Timestamp + Duration` and `Duration + Duration` should of course share code, but that is an implementation detail.

Reply:
I changed Timestamp to depend on Duration.